lib/components/Editor/utils.js

```diff
@@ -39,3 +40,19 @@ export const setInitialPosition = editor => {
   transaction.setSelection(endPosition);
   view.dispatch(transaction);
 };
+
+export const isEditorOverlaysActive = () => {
+  const isTippyContentActive = !isEmpty(
+    document.getElementsByClassName("tippy-content")
+  );
+
+  const isOverlaysActive = !isEmpty(
+    document.getElementsByClassName("ne-media-uploader")
+  );
+
+  const isEmbedModalOpen = !isEmpty(
+    document.getElementsByClassName("ne-embed-modal")
+  );
+
+  return isTippyContentActive || isOverlaysActive || isEmbedModalOpen;
```

Comment:
This can be simplified further:

```suggestion
  const isOverlaysActive = !isEmpty(
    document.querySelector(".ne-media-uploader,.ne-embed-modal")
  );

  return isTippyContentActive || isOverlaysActive;
```

Please make this change and verify it is working as expected.